test/rubygems/test_gem_specification.rb

```diff
@@ -2957,8 +2957,7 @@ def test_validate_license
     end
 
     assert_match <<-warning, @ui.error
-WARNING:  licenses is empty, but is recommended.  Use a license identifier from
-http://spdx.org/licenses or 'Nonstandard' for a nonstandard license.
+WARNING:  licenses is empty, but is recommended.  Use a license identifier from http://spdx.org/licenses or 'Nonstandard' for a nonstandard license.
```

Comment:
why has the wrapping on this changed?

Reply:
I found the old wrapping weird looking and arbitrary. I found out about it when writing [this test](https://github.com/rubygems/rubygems/pull/2332/files#diff-7c25fba4c6ec502e025345c5f4d6a5edR89), where I was expecting to check for one warning per line until no more output, but I was getting the warning broken into two lines.

I agree it's probably a discussion for other PR, so I'll undo it!

Is history editing ok, by the way, or should I add the revert and test fixes as separate commits?

Reply:
Reverted as a separate commit, I can cleanup history later.